app/views/email/edit.html.erb

```diff
@@ -7,6 +7,7 @@
       <%= f.govuk_error_summary %>
       <%= f.govuk_email_field(
             :email, 
+            autofocus: true,
```

Comment:
Not sure we should use `autofocus`, the user's focus should just follow the page naturally.

Reply:
Ok. 👍